test/yummy/config_test.clj

```diff
@@ -147,3 +147,68 @@
     (testing "slurp loading"
              (is (= {:a "hello"}
                     cfg)))))
+
+
+(def conform-input-ok
+  ["date-year: '2019'"
+   "date-date: '2019-12-31'"
+   "date-full: '2019-12-31T11:59:59'"])
```

Comment:
I insist that dates are bad example as this is fully supported by the YAML specifications.

```clj
(yaml/parse-string "{year: 2019, date: 2019-12-31, full: 2019-12-31T11:59:59Z}")
#ordered/map ([:year 2019]
              [:date #inst "2019-12-31T00:00:00.000-00:00"]
              [:full #inst "2019-12-31T11:59:59.000-00:00"])
```